docs/Initial-Setup/Setting-up-your-Manager-Instance.rst

```diff
@@ -20,44 +26,50 @@ right corner, ensure that the correct region is selected.
 
 To launch a manager instance, follow these steps:
 
-1. From the main page of the EC2 Management Console, click
-   ``Launch Instance``. We use an on-demand instance here, so that your
+#. From the main page of the EC2 Management Console, click
+   *Launch Instance*. We use an on-demand instance here, so that your
```

Comment:
```suggestion
    *Launch Instance ▼* button and click *Launch Instance* in the dropdown that appears. We use an on-demand instance here, so that your
```